### DISCUSSION TITLE
Limit number of warning messages

### DISCUSSION THREAD

**Discussion by ajgringo619** [nicolargo/glances#2554] [Q&A]
Is there a way to limit the number of warnings that appear on the bottom of the processes area? I don't mind seeing a few of the latest ones, but sometimes I get 10+ filling the screen when I'm doing disk-intensive work (compiling, running VMs/games, etc.). Even a timeout before a message is removed would be great.

**Comment by nicolargo** [comment]
Hi @ajgringo619 

for the moment, configuration of the alert plugin is limited.

I see two solutions with the currents Glances features:

- disable the alert plugin (--disable-plugin alert) 
- from the configuration file set the warning thresholds to highers values ([cpu] section)

If you have any proposal concerning new useful features for the Glances alert plugin, please open an new issue with a precise description.

Nicolas

**Reply by ajgringo619** [reply]
OK, I added a feature request. Let me know if you need any more details.
